matlab_simulation/05-estimation/fastSLAM/README.txt

```diff
@@ -0,0 +1,11 @@
+1. ROOT is run_fastSLAM.m. The code execution starts from here. 
+
+2. Uncomment/comment the fastSLAM or fastSLAM2 functions calls in run_fastSLAM to run individual code.
+
+3. the staticMap.m can be replaced for newFeatures.
+
+4. set the randMapFlag=1 for randomizing map 0: to load static map.
+
+5. All the functions contain descriptions of the input and output parameters
+
+NOTE: This example is created as a part of Assignment:3 in ME-780 Autonomous Mobile Robotics(Winter'16) course taught by Prof.Steven Waslander at the University of Waterloo. Reuse and redistribution of this code must be done after prior approval from him and the author.
```

Comment:
Haha...I had earlier read a similar ReadMe file with such a disclaimer/Important Note. This is just an attempt to fall into the habit of including a ReadMe/ license,but do feel free to use.:P